packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -147,6 +147,7 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
         itExpects("Send ops fails for tombstoned datastores in summarizing container loaded after sweep timeout",
         [
             { eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded" },
+            { eventName: "fluid:telemetry:FluidDataStoreContext:Tombstone_DataStore_Changed" },
```

Comment:
Interesting that this is Changed but the Summarizer one is Loaded, can you explain that?

Reply:
So we have the tombstoned datastore in memory and we're sending an op, so we are throwing here. In the summarizer, we load the tombstoned datastore, but the op never makes it over the wire, so we don't get sweepreadyobject_changed because the op never makes it. Even when we are receiving the op, we throw an error before we call inactiveNodeUsed